packages/picasso/src/Notes/styles.ts

```diff
@@ -0,0 +1,27 @@
+// import { PicassoProvider } from '@toptal/picasso-provider'
+import { Theme, createStyles } from '@material-ui/core/styles'
+
+// PicassoProvider.override(({ palette }: Theme) => ({}))
+
+export default ({ palette }: Theme) =>
+  createStyles({
+    root: {
+      borderRadius: '0.4rem',
+      border: `1px solid ${palette.grey.lighter}`,
+      position: 'relative',
+      overflow: 'hidden',
+      '&:before': {
+        background: palette.yellow.main,
+        display: 'block',
+        height: '100%',
+        left: 0,
+        position: 'absolute',
+        content: '""',
```

Comment:
I think it should work

```suggestion
        content: '',
```

Reply:
I was surprised too, but sadly it doesn't work with only `''` 

https://stackoverflow.com/questions/40965977/cant-target-before-pseudo-selector-in-jss